PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -333,7 +333,7 @@ systemctl mask apt-daily.service
 systemctl mask apt-daily-upgrade.timer
 
 #/etc/sysctl.conf | Check for a previous entry before adding this
-echo -e "vm.swappiness=1" >> /etc/sysctl.conf
+echo -e "vm.swappiness=1" >> /etc/sysctl.d/97-dietpi.conf
```

Comment:
If we use ```>``` here, it will overwrite the file with the entry. ```>>``` will append to end of existing file, if it exists (multiple entries if command used again)

Reply:
Oh important point, thus > is necessary here, or touch filename first, if we add something earlier/optional.